hw/lint/lint.mk

```diff
@@ -0,0 +1,44 @@
+####################################################################################################
+## Copyright lowRISC contributors.                                                                ##
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.                       ##
+## SPDX-License-Identifier: Apache-2.0                                                            ##
+####################################################################################################
+.DEFAULLT_GOAL := all
```

Comment:
typo?

Reply:
this was present in `sim.mk` as well - corrected.